spec/legacy/openstack_handle/handle_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'manageiq/providers/openstack/legacy/openstack_handle/handle'
```

Comment:
I will check it and fix integration of old tests into new provider repo in ongoing PR.